ui/src/message_popup/popup_question.jsx

```diff
@@ -118,19 +154,29 @@ export default React.createClass({
           student={student}
           question={question}
         />
-        {showResponse && <ResponseRenderer
-          scaffolding={scaffolding}
-          question={question}
-          limitMs={limitMs}
-          onResponseSubmitted={this.onResponseSubmitted}
-          onLogMessage={this.logData}
-        />}
+        {allowResponding && this.renderResponse()}
```

Comment:
This is coming from the interaction between `VelocityTransitionGroup` in the experience page, and the Material UI `TextField`.  Since this is being animated, the root `TextField` element is mounted into the DOM before it is visible and has its full size.  The `EnhancedTextarea` inside the `TextField` component looks at the DOM to see how big it should be, and sizes itself based on that (presumably to lay out the different hint and float text labels just so).  On the initial render, this code looks at the DOM and sees it has 0 height since it's not visible yet.  After the first text is entered, `TextField` runs its layout code again and then realizes it can take up more space since the element has height now, and sizes it correctly.

I worked around this by fixing the height of the `TextField` component to be correct for two rows heigh at this font size.  In general, animating components in and out often means you have to fix some parts of the layout or sizing, so while this isn't great it seems like a reasonable workaround.  The only real cost is having to update the sizing and number of rows, and I called that out with a comment here.